frontend/src/common/enums/form/events.ts

```diff
@@ -0,0 +1,6 @@
+enum Events {
```

Comment:
```suggestion
enum FormEvent {
```
don't forget about filename